src/internal-server/app/Controllers/Http/UserController.js

```diff
@@ -304,25 +299,17 @@ class UserController {
         .rows.length > 0
     );
 
-    if (
-      workspace.services &&
-      typeof workspace.services === 'string' &&
-      workspace.services.length > 0
-    ) {
-      workspace.services = JSON.parse(workspace.services);
+    if (workspace.services) {
+      workspace.services = convertToJSON(workspace.services);
     }
     const services =
       workspace.services && typeof workspace.services === 'object'
         ? workspace.services.map(
             oldServiceId => serviceIdTranslation[oldServiceId],
           )
         : [];
-    if (
-      workspace.data &&
-      typeof workspace.data === 'string' &&
-      workspace.data.length > 0
```

Comment:
Same here (?)

Reply:
that's already handled in the `convertJSON` method